### PR TITLE
[GlobalPlanner] Add support for multicamera in global_planner

### PR DESCRIPTION
**Pull request by Jaeyoung-Lim**
This PR adds support for multicamera to `global_planner` with mostly the same interface with the local_planner. The pointcloud is republished from global_planner so that it can be consumed in a single topic to construct the Octomap.

This has been tested in SITL, with the `iris_depth_camera_3` model.

To launch the 3camera model, 
```
roslaunch global_planner global_planner_sitl_3cam.launch
```

Screenshot from 2019-06-11 13-21-42

### CODE REVIEW

File: global_planner/launch/global_planner_octomap.launch [hunk 1]
```diff
@@ -12,7 +12,8 @@
         <param name="start_pos_y" value="$(arg start_pos_y)" />
         <param name="start_pos_z" value="$(arg start_pos_z)" />
         <param name="world_name" value="$(find avoidance)/sim/worlds/$(arg world_file_name).yaml" />
-        <remap from="/camera/depth/points" to="$(arg point_cloud_topic)" />
+        <rosparam param="pointcloud_topics" subst_value="True">$(arg pointcloud_topics)</rosparam>
+        <!-- <remap from="/camera/depth/points" to="$(arg point_cloud_topic)" /> -->
```
Comment (mrivi) [inline global_planner/launch/global_planner_octomap.launch]: can we get rid of commented lines?

File: global_planner/launch/global_planner_sitl_3cam.launch [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    <arg name="world_path" default="$(find avoidance)/sim/worlds/$(arg world_file_name).world" />
+    <arg name="pointcloud_topics" default="[/camera_front/depth/points,/camera_left/depth/points,/camera_right/depth/points]"/>
+    <arg name="start_pos_x" default="0.5" />
+    <arg name="start_pos_y" default="0.5" />
+    <arg name="start_pos_z" default="3.5" />
+
+    <!-- Define a static transform from a camera internal frame to the fcu for every camera used -->
+    <node pkg="tf" type="static_transform_publisher" name="tf_local_origin"
+          args="0 0 0 0 0 0 world local_origin 10"/>
+    <node pkg="tf" type="static_transform_publisher" name="tf_camera"
```
Comment (mrivi) [inline global_planner/launch/global_planner_sitl_3cam.launch]: is this static transform necessary?

Comment (Jaeyoung-Lim) [reply global_planner/launch/global_planner_sitl_3cam.launch]: I am having a hard time understanding why, but deleting it results breaking the system

File: global_planner/launch/global_planner_octomap.launch (hunk 1 above)
Comment (Jaeyoung-Lim) [reply global_planner/launch/global_planner_octomap.launch]: Sure!

File: global_planner/launch/global_planner_sitl_3cam.launch (hunk 2 above)
Comment (Jaeyoung-Lim) [reply global_planner/launch/global_planner_sitl_3cam.launch]: I think we can delete them

**Comment by Jaeyoung-Lim** [comment]
@mrivi I have addressed the comments!

**Pull request merged**